client/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,101 @@
+// version1 my code
+import { useState } from "react";
+import SearchBar from "./SearchBar";
+// import { debouncedSearchQuestions } from "../helpers/questions";
+import { debouncedSearchQuestions } from "../questions";
```

Comment:
what is this and where is it coming from? the client and server folders are totally separate and do not know about each other